src/_forms.scss

```diff
@@ -60,239 +150,306 @@ category: Common
     margin-bottom: 0;
-    margin-top: $gs-baseline/2;
 }
 
-.form-field__note--left {
-    float: left;
+.form-field--error {
+    .label {
+        color: $pasteup-forms-error-label-colour;
+    }
+
+    .input-text,
+    .input-text:focus,
+    .input-textarea,
+    .input-textarea:focus {
+        border-color: $pasteup-forms-error-input-border-colour;
+    }
 }
 
-.form-field__note--right {
-    float: right;
-    margin-left: $gs-gutter;
+.form-field__error-message {
+    @include fs-data(3);
+    color: $pasteup-forms-error-message-colour;
+    margin-top: rem(2px);
+    margin-bottom: rem($gs-baseline / 2);
 }
 
-.form-fields__inline {
-    ul {
-        margin: 0;
-        padding: 0;
+.form-note {
+    @include fs-textSans(1);
+    color: $pasteup-forms-note-colour;
+    padding-top: rem($gs-baseline / 2);
+    margin-bottom: rem(8px);
+}
 
-        .form-field {
-            display: inline-block;
-        }
+.form-note--left {
+    @include mq(tablet) {
+        float: left;
     }
+}
 
-    .form-fields__heading {
-        @include fs-bodyHeading(3, true);
-        margin: 0 0 $gs-baseline;
-        display: block;
+.form-note--right {
+    @include mq(tablet) {
+        text-align: right;
+        margin-left: rem($gs-gutter);
     }
 }
 
-.form-fields--date-of-birth .form-field {
-    margin: 0 18px 0 0;
+.form-note--underneath {
+    margin-top: rem($gs-baseline / 2);
+}
+
+.form-legal {
+    @include fs-textSans(3);
+    color: $pasteup-forms-note-colour;
 }
 
+// Form labels and inputs
+// ============================================================================
+
 .label {
+    @include fs-bodyCopy(2);
+    color: $pasteup-forms-label-colour;
     cursor: pointer;
     display: block;
-    margin-bottom: 6px;
 }
 
-.text-input,
-.textarea {
-    border: 1px solid $pasteup-forms-fields-border-colour;
-    @include box-shadow(none);
-    @include box-sizing(border-box);
-    color: $pasteup-forms-fields-colour;
+.label--inline {
     display: inline-block;
-    font-family: $pasteup-forms-fields-font-family;
-    padding: 8px 8px 7px;
-    font-size: 16px;
-    line-height: 1.4;
+}
+
+.input-text,
+.input-textarea {
+    color: $pasteup-forms-input-colour;
+    border: 1px solid $pasteup-forms-input-border-colour;
+    @include f-textSans;
+    @include font-size(16px, 22px);
+    padding: rem(8px) rem(8px) rem(7px);
     outline: none;
-    @include border-radius(0);
+    @include box-sizing(border-box);
     width: 100%;
     -webkit-appearance: none;
 
     &:focus {
-        border-color: $pasteup-forms-fields-border-colour-focus;
+        border-color: $pasteup-forms-input-border-colour-focus;
     }
 
     @include mq(tablet) {
-        font-size: 14px;
+        font-size: rem(14px);
     }
 }
 
-.textarea {
+.input-text--small {
+    width: 30%;
+}
+
+.input-textarea {
     resize: vertical;
 }
 
-.textarea--no-resize {
-    min-height: ($gs-baseline/3)*20;
-    resize: none;
+.input-textarea--long {
```

Comment:
`input-textarea--deep` instead of `--long`, also if we're going to rely on `gs-` using `gs-height()` is probably better than large multiples of baseline.